pkg/utils/syscalls/amd64/syscalls.go

```diff
@@ -0,0 +1,968 @@
+package amd64
```

Comment:
I think it's better to have these two files (amd64/syscalls.go and arm64/syscalls.go) on the same folder, then we can use build tags to only compile the one that is relevant for the current architecture. Something like for this file:

```
//go:build linux amd64
// +build linux amd64
```